packages/react-composites/src/composites/MeetingComposite/ChatButtonLogicWrapper.tsx

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { ChatMessage } from '@azure/communication-chat';
+import React, { useState } from 'react';
+import { useEffect } from 'react';
+import { ChatAdapter } from '../ChatComposite';
+import { ChatButton, ChatButtonProps } from './ChatButton';
+
+/**
+ * @private
+ */
+export type ChatButtonLogicWrapperProps = {
+  chatButtonProps: ChatButtonProps;
+  chatAdapter: ChatAdapter;
+  showChat: boolean;
```

Comment:
`showChat` makes it sound like this component shows chat.

How about `isChatPaneVisible`, because that's what the component wants to know.